src/components/CurrentTime/CurrentTime.jsx

```diff
@@ -1,7 +1,19 @@
 import classes from './CurrentTime.module.scss';
+import { useState, useEffect } from 'react';
+import { getFormatedCurrentTime } from '../../utils/dateTimeUtils';
 
 function CurrentTime() {
-  return <div className={classes.currentTime}>17:00</div>;
+  const [currentTime, setCurrentTime] = useState(() => getFormatedCurrentTime());
```

Comment:
if i am right u can wtite just getFormatedCurrentTima

Reply:
No. Every rerenders this functions will call twice. You can check it.

Reply:
okey, sorry

Reply:
Actually I agree with @DmitriyKrivovyaz. You should be able to do that.

Reply:
Sorry I was wrong) @DmitriyKrivovyaz was right!